openslides_backend/action/actions/committee/update.py

```diff
@@ -21,5 +27,69 @@ class CommitteeUpdateAction(UpdateAction):
             "manager_ids",
             "forward_to_committee_ids",
             "receive_forwardings_from_committee_ids",
+            "organisation_tag_ids",
         ]
     )
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if has_organisation_management_level(
+            self.datastore, self.user_id, OrganisationManagementLevel.SUPERADMIN
+        ):
+            return
+
+        manager_ids = None
+        if "manager_ids" in instance:
+            manager_ids = instance["manager_ids"]
+        else:
+            committee = self.datastore.get(
+                FullQualifiedId(self.model.collection, instance["id"]), ["manager_ids"]
+            )
+            manager_ids = committee.get("manager_ids")
+        if manager_ids is None:
+            manager_ids = []
+        is_manager = self.user_id in manager_ids
+        can_manage_organisation = has_organisation_management_level(
+            self.datastore,
+            self.user_id,
+            OrganisationManagementLevel.CAN_MANAGE_ORGANISATION,
+        )
+        if (
+            any(
+                [
+                    field in instance
+                    for field in [
+                        "name",
+                        "description",
+                        "template_meeting_id",
+                        "default_meeting_id",
+                    ]
+                ]
+            )
+            and not is_manager
+        ):
+            raise PermissionDenied("Not manager.")
+        if (
+            any(
+                [
+                    field in instance
+                    for field in [
+                        "member_ids",
+                        "manager_ids",
+                        "forward_to_committee_ids",
+                        "receive_forwardings_from_committee_ids",
+                    ]
+                ]
+            )
+            and not can_manage_organisation
+        ):
+            raise PermissionDenied(
+                f"Missing {OrganisationManagementLevel.CAN_MANAGE_ORGANISATION}"
+            )
+        if (
+            "organisation_tag_ids" in instance
+            and not is_manager
+            and not can_manage_organisation
+        ):
+            raise PermissionDenied(
+                f"Missing {OrganisationManagementLevel.CAN_MANAGE_ORGANISATION} and not manager."
```

Comment:
see above

Reply:
Cannot change this, because of the _and not manager_